cmd/operator-sdk/generate/csv.go

```diff
@@ -38,6 +38,7 @@ type csvCmd struct {
 	updateCRDs     bool
 	defaultChannel bool
 	makeManifests  bool
+	interactive    bool
```

Comment:
Failing in the CI. See: https://travis-ci.org/github/operator-framework/operator-sdk/jobs/681686553#L977
When we change the flags of commands we need to run `make generate`. See its target in the makefile. It will call the gen-doc to update the docs with the changes made.